src/GslibIO.jl

```diff
@@ -13,6 +13,17 @@ using GeoStatsBase
 # type aliases
 const Array2or3{T} = Union{AbstractArray{T,2},AbstractArray{T,3}}
 
+# legacy specification of the GSLIB format
+# (see documentation here: http://www.gslib.com/gslib_help/format.html)
+# Note that in this context variables can be
+# actual attributes as well as coordinates
+struct LegacySpec
+  header::String           # Content of the first line (documentation only)
+  nvars::Int               # An integer with the number of variables (first integer in the second line)
+  varnames::Vector{Symbol} # The variable names as Symbol in the following `nvars` lines
+  data::AbstractMatrix    # The data (a column represents one variable). It should be size of (N, `nvars`)
```

Comment:
So here we want a concrete type, probably Matrix{Float64}, also please align the comment sign.